easy_phi/app.py

```diff
@@ -80,6 +91,7 @@ def prepare(self):
         if err:
             self.set_status(400)
             self.finish({'error': err})
+            return
```

Comment:
My pull request to dev contains this change already. Please make sure 'return' is not written twice when you will be merging with dev